packages/app-admin-ui/components/index.js

```diff
@@ -1,5 +1,6 @@
 export { ListProvider, useList } from '../client/providers/List';
 export { default as ListPage, ListLayout } from '../client/pages/List/index';
+export { ItemProvider, useItem } from '../client/providers/Item';
```

Comment:
@MadeByMike exporting this is needed to use the item information in custom Hook, also the ItemProvider can be used to pass down further different item if needed in nested component inside the hook.